src/data/ores/index.js

```diff
@@ -4,3 +4,4 @@ export { goldOre } from './goldOre'
 export { ironOre } from './ironOre'
 export { silverOre } from './silverOre'
 export { stone } from './stone'
+export { saltRock } from './salt-rock'
```

Comment:
the "salt-rock" filename doesn't match the casing convention of the others, is that on purpose?

Reply:
Whoops! That was not intentional. This is fixed in https://github.com/jeremyckahn/farmhand/commit/3019b2feedec61ab2ae85fe87f791bf821268944.

Thanks for raising this!